[libtool] Remove --disable-static in build step

Would it be possible to change 
```
system "./configure", "--disable-static", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}"
``` 
to 
```
system "./configure", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}"
```
in the 2.4.6-3 `libtool` package ? This breaks the new neovim 0.4.2 compilation. What would be the impact ?

Cf [https://github.com/neovim/neovim/issues/11198](https://github.com/neovim/neovim/issues/11198)

I have just tried a fresh install. I am facing the followinf problem when install script installs `libtool`. It cannot find `libfl.so`

```
make[1]: Entering directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
Making all in .
make[2]: Entering directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
  CC       libltdl/loaders/dlopen.lo
  CC       libltdl/lt__strl.lo
  CC       libltdl/loaders/libltdl_libltdl_la-preopen.lo
  CC       libltdl/libltdl_libltdl_la-lt__alloc.lo
  CC       libltdl/libltdl_libltdl_la-lt_dlloader.lo
  CC       libltdl/libltdl_libltdl_la-lt_error.lo
  CC       libltdl/libltdl_libltdl_la-ltdl.lo
  CC       libltdl/libltdl_libltdl_la-slist.lo
  CCLD     libltdl/dlopen.la
ar: error while loading shared libraries: libfl.so.2: cannot open shared object file: No such file or directory
make[2]: *** [Makefile:1112: libltdl/dlopen.la] Error 127
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
make[1]: *** [Makefile:1606: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
make: *** [Makefile:995: all] Error 2
libtool failed to install: `make V=0 -j2` exited with 2
```

--- a/packages/augeas.rb
+++ b/packages/augeas.rb
@@ -8,8 +8,16 @@ class Augeas < Package
   source_sha256 '321942c9cc32185e2e9cb72d0a70eea106635b50269075aca6714e3ec282cb87'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/augeas-1.12.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/augeas-1.12.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/augeas-1.12.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/augeas-1.12.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '8c77e2c9e36dff045129f4e196b72051e7e085c612d43fe0f3f3f8d825c9d3ad',
+     armv7l: '8c77e2c9e36dff045129f4e196b72051e7e085c612d43fe0f3f3f8d825c9d3ad',
+       i686: 'ef2cc58fb25c5273ba50adfbbf6c577b2c13b79ecb325da15367965847dfba0d',
+     x86_64: 'fb85f457590f5b10d786e80b1a177f0942628438de38f64f7ae15856ee753b2c',
   })
 
   def self.build

--- a/packages/irssi.rb
+++ b/packages/irssi.rb
@@ -8,8 +8,16 @@ class Irssi < Package
   source_sha256 '6727060c918568ba2ff4295ad736128dba0b995d7b20491bca11f593bd857578'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/irssi-1.2.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/irssi-1.2.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/irssi-1.2.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/irssi-1.2.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'cef7c8a7eae8b5c8b3436808ba40d7ee47b6bee964accc6869ccc31d0bbd0c48',
+     armv7l: 'cef7c8a7eae8b5c8b3436808ba40d7ee47b6bee964accc6869ccc31d0bbd0c48',
+       i686: '2b55ec2d8ee5db531e350efc6e64e2e64c5b3a980bc1cf491d79d76cb873bf80',
+     x86_64: 'a507ca4d76b27659d03e421831da77ed161126ed7d8321cdd753fd4b45bd7536',
   })
 
   depends_on 'glib' => :build

--- a/packages/libpcap.rb
+++ b/packages/libpcap.rb
@@ -8,16 +8,26 @@ class Libpcap < Package
   source_sha256 '635237637c5b619bcceba91900666b64d56ecb7be63f298f601ec786ce087094'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.9.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.9.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.9.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.9.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '6ed603b0fcfc2262772ab55efaf2ac4aca927131d67e796d9d7b5ebdb4a13646',
+     armv7l: '6ed603b0fcfc2262772ab55efaf2ac4aca927131d67e796d9d7b5ebdb4a13646',
+       i686: '9eac0cd730345403835a085f5acd9b1648a2610c4cf1f2ae2c3af988d4c28499',
+     x86_64: '475bcac8bcd6f24765c68151500470141d5faf85fa89879612d4a0336a43e0ef',
   })
 
+  depends_on 'libnl3'
+
   def self.build
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
-    system "make"
+    system "CFLAGS='-I#{CREW_PREFIX}/include/libnl3' ./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
+    system 'make'
   end
 
   def self.install
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
 end

--- a/packages/libressl.rb
+++ b/packages/libressl.rb
@@ -8,8 +8,16 @@ class Libressl < Package
   source_sha256 'df7b172bf79b957dd27ef36dcaa1fb162562c0e8999e194aa8c1a3df2f15398e'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libressl-3.0.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libressl-3.0.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libressl-3.0.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libressl-3.0.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '42bcf153de5343a4c131424d4e09ca7f4f8e1749a0fc2e0f632f7992ba2351b3',
+     armv7l: '42bcf153de5343a4c131424d4e09ca7f4f8e1749a0fc2e0f632f7992ba2351b3',
+       i686: 'f613572c9f9c91b79b09d008fa387ecb393be9b60d37f837ea6cdc74cdf9633b',
+     x86_64: 'a2fce2036a54b1b3ca597d4fb3ee358c9fb7b3b54ea1e3a944fc7eb35a6f8e81',
   })
 
   def self.build

--- a/packages/libtool.rb
+++ b/packages/libtool.rb
@@ -8,8 +8,16 @@ class Libtool < Package
   source_sha256 'e3bd4d5d3d025a36c21dd6af7ea818a2afcd4dfc1ea5a17b39d7854bcd0c06e3'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-4-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '4d83d3c0d26b8aef3fba0d0d42feaa3fda736652a82ecfd24f4cdce384dc2b0c',
+     armv7l: '4d83d3c0d26b8aef3fba0d0d42feaa3fda736652a82ecfd24f4cdce384dc2b0c',
+       i686: '6eb2da330f7d23c34f41f5f7d94e1abc2f4b4feeb59d4c3886bfb00c8d34b4fb',
+     x86_64: 'c0de77f4eeee3d662c9575b69125fa2778fe623ffad54de5ed296a01622e3724',
   })
 
   depends_on 'm4'

--- a/packages/micro.rb
+++ b/packages/micro.rb
@@ -18,8 +18,16 @@ class Micro < Package
   end
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/micro-1.4.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/micro-1.4.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/micro-1.4.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/micro-1.4.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '305c70b8ae3ff6e8e80baf8d9af09db29b11fd3e40d381c787ee1f88406cc166',
+     armv7l: '305c70b8ae3ff6e8e80baf8d9af09db29b11fd3e40d381c787ee1f88406cc166',
+       i686: 'f92d08c76dc125b24674754b3a29ae94bccbfe2bcfca68ae21384e2215054a74',
+     x86_64: '64028544e15314a6a576521a7d21e156b9e82eda91fda40033ec19e2c75d1881',
   })
 
   def self.install

--- a/packages/tcpflow.rb
+++ b/packages/tcpflow.rb
@@ -8,8 +8,14 @@ class Tcpflow < Package
   source_sha256 '20abe3353a49a13dcde17ad318d839df6312aa6e958203ea710b37bede33d988'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/tcpflow-1.5.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/tcpflow-1.5.0-chromeos-armv7l.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/tcpflow-1.5.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '080ce34873a9d15239a61117c58124946187278d39f0c432928e5dd70a919d59',
+     armv7l: '080ce34873a9d15239a61117c58124946187278d39f0c432928e5dd70a919d59',
+     x86_64: 'c30c853d77b7111d2f03af13b605e430d24f16fa4ea4349288ebe63488fd4fc6',
   })
 
   depends_on 'boost'

